src/agent/onefuzz/examples/test-input.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use std::path::PathBuf;
+
+use anyhow::Result;
+use onefuzz::input_tester::Tester;
+use structopt::StructOpt;
+
+#[derive(Debug, PartialEq, StructOpt)]
+struct Opt {
+    #[structopt(short, long)]
+    pub exe: PathBuf,
+
+    #[structopt(short, long)]
+    pub options: Vec<String>,
+
+    #[structopt(short, long)]
+    pub setup_dir: PathBuf,
+
+    #[structopt(short, long)]
+    pub input: PathBuf,
+
+    #[structopt(long)]
+    pub check_asan_log: bool,
+
+    #[structopt(long)]
+    pub check_asan_stderr: bool,
+
+    #[structopt(long)]
+    pub no_check_debugger: bool,
+
+    #[structopt(short, long, long_help = "Timeout (seconds)", default_value = "5")]
+    timeout: u64,
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let opt = Opt::from_args();
+
+    let env = Default::default();
+    let tester = Tester::new(&opt.setup_dir, &opt.exe, &opt.options, &env);
```

Comment:
can we make the `setup-dir` optional and take the directory of the exe as the default value

Reply:
I'd considered the same thing. Guess I should probably actually do it, then 😄